project_utils/cpp/generation/SysIdSetup.cpp

```diff
@@ -14,6 +14,8 @@
 #include <wpi/StringMap.h>
 #include <wpi/StringRef.h>
 
+#include "sysid/Util.h"
+
```

Comment:
Looks like the program crashes because it can't import this. Should we create a `Util.h` for `project_utils` (code duplication) or link the sysid files in the build.gradle

Reply:
All it needs is the stringization macro. We can put those two lines anywhere.